adapter2/src/debug_session.rs

```diff
@@ -360,9 +360,13 @@ impl DebugSession {
                     Err(Error::Internal("Not implemented.".into()))
                 }
             },
-            // A special case for DebugClient, which omits "disconnect" arguments.
+            // Special cases for DebugClients, which omit optional arguments.
             Command::Unknown { ref command } if command == "disconnect" =>
                 self.handle_disconnect(None).map(|_| ResponseBody::disconnect),
+            Command::Unknown { ref command } if command == "configurationDone" =>
+                self.handle_configuration_done().map(|_| ResponseBody::configurationDone),
+            Command::Unknown { ref command } if command == "threads" =>
```

Comment:
But according to DAP spec, "threads" is not supposed to have any arguments...

Reply:
Looks like Theia just always adds "arguments":{}.   Ok, I guess there's no harm in tolerating a minor spec violation.